static/ankersrv.css

```diff
@@ -117,3 +117,26 @@ code {
     border: 0 none;
     padding: 0;
 }
+
+.fullscreen-emulate {
```

Comment:
Should this be `#vmain.fullscreen-emulate`?

Otherwise, this applies to `body`, `header`, `footer` and `#vmain`. We hide header and footer, but this makes these rules apply to both `body` and `#vmain`, no?

Reply:
I'm pretty certain it overrides but I can test to see if it is inheriting the un-specified class.

Reply:
Tested and pushed the change :) 

✅